run_test.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -x
+
+# Run sanity tests for sonic-mgmt-common.
+# Assumes sonic-mgmt-common is already compiled and all dependencies
+# are installed.
+
+STATUS=0
+DEBDIR=$(realpath debian/sonic-mgmt-common)
+
+OUTPUT_DIR=./
+DETAILED_COV=n
+SKIP_BUILD=n
```

Comment:
I don't see DETAILED_COV and SKIP_BUILD used anywhere. Is this an incomplete script?